### PR TITLE
Create Purdue_Sharma.yaml

### PR DESCRIPTION
**Pull request by showmic09**

### CODE REVIEW

**Comment by matyasselmeci** [comment]
@opensciencegrid/project-office permission to add the organization "[Purdue University Fort Wayne](https://www.pfw.edu/)"?

@showmic09, can you rename the file so it starts with "PFW_" instead of "Purdue_"? We already use the Purdue_ prefix for projects from [Purdue University](https://www.purdue.edu/).

**Comment by showmic09** [comment]
@matyasselmeci will do

**Comment by showmic09** [comment]
@matyasselmeci Done

**Comment by jstathas** [comment]
Purdue University Fort Wayne<https://www.pfw.edu/> is approved.

From: Matyas Selmeci ***@***.***>
Date: Wednesday, May 1, 2024 at 9:33 AM
To: opensciencegrid/topology ***@***.***>
Cc: Stathas, Janet ***@***.***>, Team mention ***@***.***>
Subject: Re: [opensciencegrid/topology] Create Purdue_Sharma.yaml (PR #3846)

@opensciencegrid/project-office<https://github.com/orgs/opensciencegrid/teams/project-office> permission to add the organization "Purdue University Fort Wayne<https://www.pfw.edu/>"?

@showmic09<https://github.com/showmic09>, can you rename the file so it starts with "PFW_" instead of "Purdue_"? We already use the Purdue_ prefix for projects from Purdue University<https://www.purdue.edu/>.

File: projects/PFW_Sharma.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Department: Department of Chemistry and Biochemistry
+Description: 'Investigating protein-protein interactions related to the human immune
+  system. Other projects include immune protein-drug interactions. We mainly use molecular
+  dynamics simulations and molecular coking methodologies for our study. '
+FieldOfScience: Chemistry
+FieldOfScienceID: '40.0511'
+InstitutionID: Unknown
```
Comment (mwestphall) [inline projects/PFW_Sharma.yaml:7]:
```suggestion
InstitutionID: 'https://osg-htc.org/iid/dgsgv9qbvgce'
```

**Review by mwestphall — APPROVED** [review]

**Pull request merged**